Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN conda install -c conda-forge google-cloud-sdk && \
     apt-get update -y && \
     apt-get install --no-install-recommends -y \
         git gcc rsync sudo patch openssh-server \
-        pciutils nano fuse socat netcat-openbsd curl rsync vim tini autossh && \
+        pciutils nano fuse socat netcat-openbsd curl rsync vim tini autossh jq && \
```

Comment:
Let's also add to Dockerfile_k8s and Dockerfile_k8s_gpu just in case?

Reply:
Sounds good!